doc/cephfs/disaster-recovery-experts.rst

```diff
@@ -99,30 +99,29 @@ also need to reset the other tables then replace 'session' with 'snap' or 'inode
 MDS map reset
 -------------
 
-Once the in-RADOS state of the file system (i.e. contents of the metadata pool)
-is somewhat recovered, it may be necessary to update the MDS map to reflect
-the contents of the metadata pool.  Use the following command to reset the MDS
-map to a single MDS:
+When the in-RADOS state of the file system (that is, the contents of the
+metadata pool) has been somewhat recovered, it may be necessary to update the
+MDS map to reflect the new state of the contents of the metadata pool.  Use the
```

Comment:
suggest "the new state of the metadata pool"

Reply:
I agree.